string fruit
int32 fruit_score
float32 xmin
float32 xmax
float32 ymin
float32 ymax
string disease "undefined"
int32 disease_score 0